tools/codemods/fix_bare_except.py

```
@@ -75,6 +75,21 @@ def replace_bare_except(text: str, file_hint: str) -> str:
     result = "\n".join(updated_lines)
     if changed:
         result = ensure_logging_import(result)
+
+        # Уберём случайные соседние дубликаты одинаковых logging.exception(...)
+        def dedupe_match(m):
+            lines = m.group(0).splitlines()
+            # Найдём первую строку с logging.exception
```

Ruff (0.13.3)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)